src/lib/utils/liquidityMining/MultiTokenLiquidityMining.json

```diff
@@ -0,0 +1,30 @@
+{
+  "week_57": [
+    {
+      "chainId": 1,
+      "pools": {
+          "0xa6f548df93de924d73be7d25dc02554c6bd66db500020000000000000000000e": [
+            {
+              "tokenAddress": "0xba100000625a3754423978a60c9317c58a424e3d",
+              "amount": 15000
+            },
+            {
+              "tokenAddress": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+              "amount": 10000
+            }
+          ]
+        }      
+    },
+    {
+      "chainId": 137,
+      "pools": {
+        "0x0297e37f1873d2dab4487aa67cd56b58e2f27875000100000000000000000002": [
+          {
+            "tokenAddress": "0x9a71012b13ca4d3d0cdc72a177df3ef03b0e76a3",
```

Comment:
@markusbkoch this token address is the BAL Polygon address - these all must be polygon token addresses, otherwise it won't work. Is that ok?